<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>666</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget_massive">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>531</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>200</number>
    </attribute>
    <row/>
    <column/>
   </widget>
   <widget class="QSpinBox" name="spinBox_count">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="count">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Кол-во:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_randomcount">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Рандомное кол-во</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_removeLast">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить последнюю ячейку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_randomValue">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Рандомное заполнение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_min">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Минимум</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_max">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>160</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Максимум</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_avg">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>160</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Среднее</string>
    </property>
   </widget>
   <widget class="QLabel" name="MinValue">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>210</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Мин знач</string>
    </property>
   </widget>
   <widget class="QLabel" name="AvgValue">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ср знач</string>
    </property>
   </widget>
   <widget class="QLabel" name="MaxValue">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Макс знач</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sortcomp">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Расчётска</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sortmonkey">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Обезъянья</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sortbooble">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>250</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Пузырёк</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sortgnom">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>300</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Гномья</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sortfast">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>250</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Быстрая</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_removeall">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить всё</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>420</y>
      <width>471</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Поиск</string>
    </property>
    <widget class="QLabel" name="lab_search">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Искомое значение:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_searchCount">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Кол-во вхождений:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_searchItemsNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Номер ячейки:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_searchCountValue">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Кол-во </string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>33</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Найти</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
